analytics_data_api/v0/views/learners.py

```diff
@@ -60,6 +60,7 @@ class LearnerView(LastUpdateMixin, CourseViewMixin, generics.RetrieveAPIView):
         Returns metadata and engagement data for the learner in JSON format.
 
             * username: The username of the enrolled learner.
+            * account_url:  URL to learner's account page.
```

Comment:
Oh, I actually thought that this should point to the learner's account page, e.g. https://courses.edx.org/u/[username].  I thought we were using this link in the learner's page on Insights.  It's good that we're clarifying this! 😄